### PR TITLE
[HD32][InContextLearning] Update Component Styling

### PR DESCRIPTION
**Pull request by annmariedoyle-shopify**
### WHY are these changes introduced?

Fixes #0000 

### How to 🎩

🖥 [Local development instructions](https://github.com/Shopify/polaris/blob/main/README.md#local-development)
🗒 [General tophatting guidelines](https://github.com/Shopify/polaris/blob/main/documentation/Tophatting.md)
📄 [Changelog guidelines](https://github.com/Shopify/polaris/blob/main/documentation/Versioning%20and%20changelog.md)

<details>
<summary>Copy-paste this code in <code>playground/Playground.tsx</code>:</summary>

```jsx
import React from 'react';
import {Page} from '../src';

export function Playground() {
  return (
    <Page title="Playground">
      {/* Add the code you want to test in here */}
    </Page>
  );
}
```

</details>

### 🎩 checklist

- [ ] Tested on [mobile](https://github.com/Shopify/polaris/blob/main/documentation/Tophatting.md#cross-browser-testing)
- [ ] Tested on [multiple browsers](https://help.shopify.com/en/manual/shopify-admin/supported-browsers)
- [ ] Tested for [accessibility](https://github.com/Shopify/polaris/blob/main/documentation/Accessibility%20testing.md)
- [ ] Updated the component's `README.md` with documentation changes
- [ ] [Tophatted documentation](https://github.com/Shopify/polaris/blob/main/documentation/Tophatting%20documentation.md) changes in the style guide
- [ ] For visual design changes, ping @ sarahill to update the Polaris UI kit

### CODE REVIEW

File: polaris-react/src/components/InContextLearning/InContextLearning.scss [hunk 1]
```diff
@@ -0,0 +1,16 @@
+.InContextLearning {
+  position: relative;
+  min-width: 250px;
```
Comment (annmariedoyle-shopify) [inline polaris-react/src/components/InContextLearning/InContextLearning.scss:3]: This number is a little arbitrary (just a number that I knew would fix the step counter, next, and back buttons) and we might want to set this more programatically and set it as an inline-style. But I think for hack days maybe this is fine?

File: polaris-react/src/components/InContextLearning/InContextLearning.tsx [hunk 2]
```diff
@@ -109,42 +99,46 @@ export function InContextLearning({children, steps, onDismiss}: Props) {
...
-                  wrap={false}
-                  distribution="fillEvenly"
-                >
-                  <Stack.Item>
+                <Header title={title} onDismiss={onDismiss} />
+                <div className={styles.TutorialContent}>
+                  {steps[currentStep].content}
+                </div>
+                <div className={styles.TutorialFooter}>
+                  <Stack
```
Comment (annmariedoyle-shopify) [inline polaris-react/src/components/InContextLearning/InContextLearning.tsx:95]: Personally, I do not like the stack component and find it hard to style and never actually does what I want it to do. I think we should look creating something custom here.

**Review by rockinrobin714 — APPROVED** [review]
LGTM

File: polaris-react/src/components/InContextLearning/InContextLearning.tsx (hunk 2 above)
Comment (alololox) [reply polaris-react/src/components/InContextLearning/InContextLearning.tsx:95]: I don't mind `stack`, and think we can do without any custom styling here to match the [mocks](https://www.figma.com/file/gAysnv8y94Dj3V4ZKoykdW/Prep-work?node-id=140%3A2898):

```jsx
<Stack alignment="center" wrap={false} distribution="fillEvenly">
  <Stack.Item>
    {hasMultipleSteps && (
      <TextStyle variation="subdued">
        {currentStep + 1} of {totalSteps}
      </TextStyle>
    )}
  </Stack.Item>
  <Stack.Item>
    <Stack
      alignment="center"
      wrap={false}
      spacing="tight"
      distribution="trailing"
    >
      {showBack && <Button onClick={handleBack}>Back</Button>}
      {showNext && (
        <Button primary onClick={handleNext}>
          Next
        </Button>
      )}
      {showClose && (
        <Button primary onClick={handleClose}>
          Got it
        </Button>
      )}
    </Stack>
  </Stack.Item>
</Stack>;
```

File: polaris-react/playground/Learner.tsx [hunk 3]
```diff
@@ -25,7 +25,11 @@ export function Learner() {
       <Page narrowWidth>
         <Layout>
           <Layout.Section>
-            <InContextLearning steps={LEARNING_STEPS} onDismiss={() => {}} />
+            <InContextLearning
+              steps={LEARNING_STEPS}
+              onDismiss={() => {}}
+              title="Shipping Tracking"
```
Comment (alololox) [inline polaris-react/playground/Learner.tsx]: Question for UX: Is the title for each tip constant or changing with the content?

Screen Shot 2022-05-11 at 8 17 10 AM
(Title here is 'Tipping')

**Review by alololox — APPROVED** [review]

**Pull request merged**